### ISSUE TITLE
BATTLEFURY DOES NOT CLEAVE!!!!!!!!

### ISSUE THREAD

**Issue by sanjaykig** [ValveSoftware/Dota2-Gameplay#14447]
### Item name

Battlefury

### Description

BATTLEFURY DOES NOT CLEAVE!!!!!!!!

Match ID: 7495399602

### Example Match ID and Timestamp

7495399602

**Comment by kabutix** [comment]
Yes please fix this. Played as PA with the persona skin, cleave is visually seen but no damage at all.
Match ID: 7495395146
